### PR TITLE
Enable filtering administrators based on the userstore in console settings page

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

https://github.com/wso2/identity-apps/assets/41533942/6d739590-9efe-4ffd-985f-25b4e7a436ef

### Related Issues
- https://github.com/wso2/product-is/issues/19043

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/console-settings/components/console-administrators/administrators-list/administrators-list.tsx [hunk 1]
```diff
@@ -322,30 +336,48 @@ const AdministratorsList: React.FunctionComponent<AdministratorsListProps> = (
...
-                onIsLoading={ setLoading }
-                selection={ selection }
-                showListItemActions={ showListItemActions }
-                showMetaContent={ showMetaContent }
-                data-componentid={ `${componentId}-table` }
-            />
+            { adminUserListFetchError
+                ? (<EmptyPlaceholder
+                    subtitle={ [ t("console:manage.features.users.placeholders.userstoreError.subtitles.0"),
+                        t("console:manage.features.users.placeholders.userstoreError.subtitles.1")     ] }
```
Comment (JayaShakthi97) [inline apps/console/src/features/console-settings/components/console-administrators/administrators-list/administrators-list.tsx:358]:
```suggestion
                        t("console:manage.features.users.placeholders.userstoreError.subtitles.1") ] }
```

File: apps/console/src/features/console-settings/components/console-administrators/console-administrators.tsx [hunk 2]
```diff
@@ -50,15 +55,63 @@ const ConsoleAdministrators: FunctionComponent<ConsoleAdministratorsInterface> =
...
 
     const { t } = useTranslation();
 
+    const {
+        data: userStoreList,
+        isLoading: isUserStoreListFetchRequestLoading
+    } = useUserStores(null);
+
+    useEffect(() => {
+        if ( userStoreList && !isUserStoreListFetchRequestLoading) {
```
Comment (JayaShakthi97) [inline apps/console/src/features/console-settings/components/console-administrators/console-administrators.tsx:68]:
```suggestion
        if (userStoreList && !isUserStoreListFetchRequestLoading) {
```

File: apps/console/src/features/console-settings/components/console-administrators/invited-administrators/invited-administrators-list.tsx [hunk 3]
```diff
@@ -236,22 +260,32 @@ const InvitedAdministratorsList: React.FunctionComponent<InvitedAdministratorsLi
...
-                onIsLoading={ setLoading }
-                selection={ selection }
-                showListItemActions={ showListItemActions }
-                showMetaContent={ showMetaContent }
-                data-componentid={ `${componentId}-table` }
-            />
+            { adminUserListFetchError
+                ? (<EmptyPlaceholder
+                    subtitle={ [ t("console:manage.features.users.placeholders.userstoreError.subtitles.0"),
+                        t("console:manage.features.users.placeholders.userstoreError.subtitles.1")     ] }
```
Comment (JayaShakthi97) [inline apps/console/src/features/console-settings/components/console-administrators/invited-administrators/invited-administrators-list.tsx:266]:
```suggestion
                        t("console:manage.features.users.placeholders.userstoreError.subtitles.1") ] }
```

**Review by JayaShakthi97 — DISMISSED** [review]

File: apps/console/src/features/console-settings/components/console-administrators/console-administrators.tsx [hunk 4]
```diff
@@ -50,15 +55,63 @@ const ConsoleAdministrators: FunctionComponent<ConsoleAdministratorsInterface> =
...
+
+            setAvailableUserStores(storeOptions);
+        }
+    }, [ userStoreList, isUserStoreListFetchRequestLoading ]);
+
     const renderSelectedAdministratorGroup = (): ReactElement => {
         switch (activeAdministratorGroup) {
             case "activeUsers":
-                return <AdministratorsList />;
+                return (<AdministratorsList
```
Comment (Achintha444) [inline apps/console/src/features/console-settings/components/console-administrators/console-administrators.tsx:108]:
```suggestion
                return (
                  <AdministratorsList
```

For readability, check other places.

Comment (pavinduLakshan) [reply apps/console/src/features/console-settings/components/console-administrators/console-administrators.tsx]: Fixed

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**